tests/regression_tests/test_ustar_cp/test_ustar_cp.py

```diff
@@ -77,7 +79,9 @@ def test_ustar_cp(testcase, expected_values, setup_folders, matlab_engine, find_
     # Step 6: Assert that the MATLAB function returns the expected exit code
     assert exitcode == expected_values[0], f"Expected exit code {expected_values[0]}, but got {exitcode}."
 
-
+    # Step 7: Assert that the reference output folder and test run output contain the same files
+    comparison = filecmp.dircmp(ref_outputs, test_outputs)
+    assert comparison, f"Expected True, but returned False"
```

Comment:
Maybe could be more specific, like "The reference output folder and test run output do not contain the same files."